mempool/mempool.go

```diff
@@ -123,6 +128,10 @@ type Config struct {
 	// to use for indexing the unconfirmed transactions in the memory pool.
 	// This can be nil if the address index is not enabled.
 	ExistsAddrIndex *indexers.ExistsAddrIndex
+
+	// RegenTemplate defines the function to use in regenerating a block
```

Comment:
I would suggest calling out '... to use to force generation of a new block template`.